HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2437,6 +2473,7 @@ def self.apply_appliances(hpxml_bldg, nbeds, eri_version, schedules_file)
       hpxml_bldg.refrigerators[0].primary_indicator_isdefaulted = true
     end
     hpxml_bldg.refrigerators.each do |refrigerator|
+      schedules_includes_fractions_multipliers = (!refrigerator.weekday_fractions.nil? || !refrigerator.weekend_fractions.nil? || !refrigerator.monthly_multipliers.nil?)
```

Comment:
Nice. If I'm following the logic correctly:
- If a partial set of coefficient schedule inputs are defined, the other coefficient inputs will be defaulted
- If a partial set of fraction schedule inputs are defined, the other fraction inputs will be defaulted
- If neither fraction or coefficient inputs are defined, coefficient inputs will be defaulted

👍